### PR TITLE
Food sad path testing

### PR DESCRIPTION
**Pull request by tnodland**
Adds several sad path tests for food routes, needs delete sad path before merge

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 config
+coverage
```
Comment (BrennanAyers) [inline .gitignore]: Smart!

File: models/food.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 module.exports = (sequelize, DataTypes) => {
   const Food = sequelize.define('Food', {
-    name: DataTypes.STRING,
-    calories: DataTypes.INTEGER
+    name: {type: DataTypes.STRING,
+           allowNull: false
+           },
+    calories: {type: DataTypes.INTEGER,
+               allowNull: false
+              }
```
Comment (BrennanAyers) [inline models/food.js]: 👍

File: test/food.spec.js [hunk 3]
```diff
@@ -54,6 +61,14 @@ describe('api', () => {
         })
     })
 
+    test("It can't create a food in the database if missing parameters", () => {
+      return request(app).post('/api/v1/foods')
+        .send('name=Test')
+        .then(response => {
+          expect(response.statusCode).toBe(500)
+        })
```
Comment (BrennanAyers) [inline test/food.spec.js]: Good start to edge cases, I think I'm going to add one for `DELETE`, and anything else we think of along the way!
